src/NuGet.Core/NuGet.Protocol/Model/IPackageSearchMetadata.cs

```diff
@@ -24,6 +24,7 @@ public interface IPackageSearchMetadata
         Uri LicenseUrl { get; }
         Uri ProjectUrl { get; }
         Uri ReadmeUrl { get; }
+        Uri ReadmeFileUrl { get; }
```

Comment:
```suggestion
        string ReadmeFileUrl { get; }
```

It's a perf mistake to use `System.Uri`.  It's expensive to calculate, and `HttpClient` has overloads that allow you to pass the URL as a string. Maybe it parses as an Uri internally, I don't know, but it also doesn't matter. This property isn't always going to be used, so we shouldn't pay the perf cost to force it to always be parsed.

I'd love to change the other ones from `Uri` to `string`, but that would be a breaking change in a package we ship. But we can avoid making the same mistake in new APIs.